lib/pages/main_menu.dart

```diff
@@ -170,6 +170,224 @@ class _MainMenuState extends State<MainMenu> {
+                                      title: Text("Blaney, Elizabeth. &quot;Police officers&#39; views of specialized intimate partner violence "
+                                          + "training.&quot; Policing: an international journal of police strategies &amp; management 33, "
+                                          + "no. 2 (2010): 354-375."),
+                                    ),
+                                    ListTile(
+                                      leading: Icon(Icons.local_library),
+                                      title: Text("Bloomberg, M. R. (2004). MEDICAL PROVIDERS’GUIDE TO MANAGING THE CARE OF DOMESTIC "
+                                          + "VIOLENCE PATIENTS WITHIN A CULTURAL CONTEXT."),
+                                    ),
+                                    ListTile(
+                                      leading: Icon(Icons.local_library),
+                                      title: Text("Buzawa, Eves, and Thomas Austin. &quot;Determining police response to domestic violence"
+                                          + "victims.&quot; Victims of Crime and the Victimization Process 36, no. 5 (2013): 16."),
+                                    ),
+                                    ListTile(
+                                      leading: Icon(Icons.local_library),
+                                      title: Text("&quot;Cycle of Violence.&quot; Shelter for Help in Emergency, Blue Cloud Studio, 2020, www."
+                                          + "shelterforhelpinemergency.org/get-help/cycle-violence#:~:text=."),
+                                    ),
+                                    ListTile(
+                                      leading: Icon(Icons.local_library),
+                                      title: Text("Decker, M. R., Frattaroli, S., McCaw, B., Coker, A. L., Miller, E., Sharps, P., ... &amp; "
+                                          + "Bennett, W. L. (2012). Transforming the healthcare response to intimate partner violence and "
+                                          + "taking best practices to scale."),
+                                    ),
+                                    ListTile(
+                                      leading: Icon(Icons.local_library),
+                                      title: Text("DeJong, Christina, Amanda Burgess-Proctor, and Lori Elis. &quot;Police officer perceptions "
+                                          + "of intimate partner violence: An analysis of observational data.&quot; Violence and victims 23, "
+                                          + "no. 6 (2008): 683-696."),
+                                    ),
+                                    ListTile(
+                                      leading: Icon(Icons.local_library),
+                                      title: Text("&quot;Federal Domestic Violence Laws.&quot; The United States Department of Justice, "
```

Comment:
Please review the rest of the sources content and resolve all occurrences of the "&quot"